python/test/unit/language/test_constexpr_type.py

```diff
@@ -0,0 +1,25 @@
+import triton.language as tl
```

Comment:
Could it be included in `test_frontend.py`?

Reply:
Done